src/server/routes/calendar.js

```diff
@@ -1,11 +1,14 @@
+'use strict';
+
 const express   = require('express'),
       router    = express.Router(),
       path      = require('path'),
       passport  = require('passport'),
       gcal      = require('google-calendar'),
       jstz      = require('jstz'), // Automatically detect timezone and initialize
       timezone  = jstz.determine();
-
+      Event     = require('../models/event'),
```

Comment:
Steven, this will throw an error. 
Remember what I just said about how the `timezone  = jstz.determine();` ending in `;` means that everything that comes after is is no longer part of the declaration of `const` on top.

Either,
1 - Convert it to a comma and then everything will be fine with your subsequent listing of Event and other parameters, or redeclare with full declaration i.e. 
`const Event = require...`

Either way always run your code. This would throw an error immediately.